opentracing/README.md

```diff
@@ -56,9 +51,11 @@ directly and manage ownership of the `opentracing.OpenTracer` explicitly.
 
 #### Creating a root Span (i.e., without a known parent)
 
+Just to show that it's not required, we don't call `AddToGoContext` this time.
+
     func xyz() {
         ...
-        sp, ctx := opentracing.StartSpan("span_name")
+        sp := opentracing.StartTrace("span_name")
```

Comment:
riffing off of the Uber terminology here (talking about the trace rather than the span).